app/blinken.js

```diff
@@ -46,8 +46,12 @@
     var self = this;
     $.getJSON(environment_url + "/cgi-bin/icinga/status.cgi?servicestatustypes=20&jsonoutput=1", function(data) {
       var service_status = data.status.service_status;
-      var critical_entries = service_status.filter(function(o) { return o.status === "CRITICAL" && o.has_been_acknowledged === false }).length;
-      var warning_entries = service_status.filter(function(o) { return o.status === "WARNING" && o.has_been_acknowledged === false }).length;
+      var critical_entries = service_status.filter(function(o) { return o.status === "CRITICAL" &&
+                                                                        o.has_been_acknowledged === false &&
+                                                                        o.in_scheduled_downtime === false }).length;
```

Comment:
These lines are a little long, I'd normally indent the function, or the function body to keep things a little more compact.